metricbeat/etc/beat.yml

```diff
@@ -43,8 +43,11 @@ metricbeat:
       #maxconn: 10
 
       #filter: ...
-      #username: name
+
+      # Redis AUTH password. Empty by default.
       #password: hello world
```

Comment:
We should probably use a password like "test1234" or "changeme" instead of one with spaces.